### DISCUSSION TITLE
SpatieMediaLibraryFileUpload visibility public or private not working on gcs , s3 is ok

### DISCUSSION THREAD

**Discussion by MilesWuCode** [filamentphp/filament#14096] [Help]
### Package

Form builder

### Package Version

v3.2.109

### How can we help you?

SpatieMediaLibraryFileUpload visibility public or private not working on gcs

somebody use GCS working?

**Comment by sertxudev** [comment]
I'm just passing through, but yesterday I was testing the visibility with Min.io and I encountered this issue: https://github.com/minio/minio/issues/8195#issuecomment-528981646

Maybe it is related to your problem, I guess GCS might not support object ACLs like Min.io, but this is just a guess, I have never used GCS.
